ext/date/date_core.c

```diff
@@ -4537,6 +4540,8 @@ date_s__parse(int argc, VALUE *argv, VALUE klass)
  * <b>Note</b>:
  * This method recognizes many forms in +string+,
  * but it is not a validator.
+ * For formats, see
+ * {Formats for Dates and Times}[https://docs.ruby-lang.org/en/master/strftime_formatting_rdoc.html].
```

Comment:
I think this should be updated to be consistent with the documentation for `Date._parse`.

Reply:
> I think this should be updated to be consistent with the documentation for `Date._parse`.

Done.